### ISSUE TITLE
ImportError: cannot import name 'run' from 'graphviz.backend'

### ISSUE THREAD

**Issue by madpower2000** [mljar/studio#75]
ImportError
cannot import name 'run' from 'graphviz.backend' (C:\Users\Max\AppData\Roaming\MLJAR-Studio\miniconda\lib\site-packages\graphviz\backend\__init__.py)

![image](https://user-images.githubusercontent.com/22916407/141810956-8e9bb78d-9991-47a0-8352-39134bfcc92f.png)

**Comment by pplonski** [comment]
Sorry, it's similar as this issue https://github.com/RasaHQ/rasa/issues/2357 - there is some problem with graphviz installer.

**Comment by madpower2000** [comment]
I can't find a solution under that thread, looks like the problem with graphviz.backend is not fixed if it is installed on Windows, right?

**Comment by madpower2000** [comment]
C:\Users\...\AppData\Roaming\MLJAR-Studio\miniconda\Lib\site-packages\dtreeviz\trees.py
Replace run for execute:
```
from graphviz.backend import execute, view
...
execute(cmd, capture_output=True, check=True, quiet=False)
```

**Comment by pplonski** [comment]
Thank you for fixes. I need to check required versions more carefully. Please leave the issue open. I would like to fix them in the next release. Thought I still not sure how the next version should look like.

Have you built any successful analysis with MLJAR Studio?

**Comment by pplonski** [comment]
This should be fixed after `dtreeviz` package update to `0.13.2` - I've updated the conda-forge channel and waiting for changes to populate.
